testsV2/postgres_test.go

```diff
@@ -56,5 +60,59 @@ func TestPostgresTcp(t *testing.T) {
 	requirePostgresVars(t)
 
 	dsn := fmt.Sprintf("user=%s password=%s database=%s sslmode=disable", *postgresUser, *postgresPass, *postgresDb)
-	proxyConnTest(t, *postgresConnName, "postgres", dsn, postgresPort, "")
+	proxyConnTest(t, []string{*postgresConnName}, "postgres", dsn, postgresPort, "")
+}
+
+// removeAuthEnvVar retrieves an OAuth2 token and a path to a service account key
+// and then unsets GOOGLE_APPLICATION_CREDENTIALS. It returns a cleanup function
+// that restores the original setup.
+func removeAuthEnvVar(t *testing.T) (*oauth2.Token, string, func()) {
```

Comment:
Is this necessary for the other 2 tests? if we specific the flag, it should override the GOOGLE_APPLICATION_CREDENTIALS

Reply:
I've removed the GOOGLE_APPLICATION_CREDENTIALS for each auth test, to ensure pass for the correct reason. If we retain the envvar, in theory the test could pass but be using the ADC.